src/data_gradients/dataset_adapters/output_mapper/dataset_output_mapper.py

```diff
@@ -82,13 +93,41 @@ def _get_labels_extractor(self, data: SupportedData) -> Callable[[SupportedData]
             f"You can find more detail about this in our documentation: https://github.com/Deci-AI/data-gradients"
         )
 
-    @staticmethod
-    def _to_torch(tensor: Union[np.ndarray, PIL.Image.Image, torch.Tensor]) -> torch.Tensor:
-        if isinstance(tensor, np.ndarray):
-            return torch.from_numpy(tensor)
-        elif isinstance(tensor, PIL.Image.Image):
-            return transforms.ToTensor()(tensor)
-        elif np.isscalar(tensor):
-            return torch.tensor(tensor)
+    @classmethod
+    def _to_torch(cls, data: Union[np.ndarray, PIL.Image.Image, torch.Tensor, str, List[Union[PIL.Image.Image, np.ndarray, str]]]) -> torch.Tensor:
+        """Convert various input types to a PyTorch tensor.
+
+        :param data: Input data to be converted. This can be:
+            - PyTorch tensor (in which case the function simply returns it)
+            - numpy ndarray
+            - PIL Image
+            - string representing the path to an image file
+            - list containing any combination of the above three types
+
+        :return: The input data converted (or simply returned if already) as a PyTorch tensor.
+        """
+        if isinstance(data, torch.Tensor):
+            return data
+        elif isinstance(data, np.ndarray):
+            return torch.from_numpy(data)
+        elif isinstance(data, PIL.Image.Image):
+            return torch.from_numpy(np.array(data))
+        elif isinstance(data, str):
+            with PIL.Image.open(data) as img:
+                return torch.from_numpy(np.array(img))
+        elif np.isscalar(data):
+            return torch.tensor(data)
+        elif isinstance(data, list):
```

Comment:
+ tuple ?
Also, I'm not sure whether this may happen or not, what if data will be ListConfig or DictConfig (OmegaConf containers)?

Reply:
Good point, I added support for `Sequence` https://github.com/Deci-AI/data-gradients/pull/199/commits/0c3dd404317be795611e8591a0fc74169e5c238f